lib/metrics.py

```diff
@@ -149,7 +148,13 @@ def _get_database_mutations(self):
             db_config = buildpackutil.get_database_config()
             with conn.cursor() as cursor:
                 cursor.execute(
-                    "select xact_commit, xact_rollback, tup_inserted, tup_updated, tup_deleted from pg_stat_database where datname = '%s';" % (
+                    "SELECT xact_commit,"
+                    "       xact_rollback,"
+                    "       tup_inserted,"
+                    "       tup_updated,"
+                    "       tup_deleted"
+                    "FROM pg_stat_database"
```

Comment:
missing space before `"` in a couple of lines here?

Reply:
yes.